source/annotations/helpers.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package annotations
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"sigs.k8s.io/external-dns/endpoint"
+)
+
+func hasAliasFromAnnotations(annotations map[string]string) bool {
+	aliasAnnotation, exists := annotations[AliasKey]
+	return exists && aliasAnnotation == "true"
```

Comment:
In Go we use "ok" as var name in these cases, please change all of them.
`exists` -> `ok`

Reply:
done